src/edu/mills/cs180a/StringUtilitiesTester.java

```diff
@@ -19,7 +24,31 @@ void isSubstringReturnsTrueForEmptySubstring() {
 
     @Test
     void isSubstringReturnsTrueForLengthOneSubstring() {
-        assertTrue(StringUtilities.isSubstring("A", "ABC"));
-        // TODO: write rest
+        assertTrue(StringUtilities.isSubstring("A", "ABC"));  
     }
+    
+    @ParameterizedTest
+    @CsvSource({"AQ, AQSRT", "ST, AFGSTAG","HXB, KXMDCBUHXB", "ABC, ABC"}) 
+    void isSubstringReturnsTrueForLengthTwoAndHigherSubstring(String substring, String text) {
+        assertTrue(StringUtilities.isSubstring(substring, text));
+    }
+    
+    @ParameterizedTest
+    @CsvSource({"ST, AQSRT", "EQ, GEHSVQ", "ABCDEF, ABC"})
+    //fails: ABCDEF, ABC expected false, but returns true
```

Comment:
This comment was nice for me to learn how you found the bug but should be modified now that it passes.